packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -596,7 +597,7 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
     super.dispose();
   }
 
-  void _toggleExpansion() {
+  Future<void> _toggleExpansion() async {
```

Comment:
Any reason to make this async? AFAICT, the futures don't need to be awaited in this method since the Semantics announcement is fire-and-forget.

Reply:
Ohh... great catch! Since it's a future i always assumed it has to be waited.